Documentation/api.md

```diff
@@ -237,6 +238,7 @@ AlertmanagerSpec is a specification of the desired behavior of the Alertmanager
 | alertmanagerConfigSelector | AlertmanagerConfigs to be selected for to merge and configure Alertmanager with. | *[metav1.LabelSelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#labelselector-v1-meta) | false |
 | alertmanagerConfigNamespaceSelector | Namespaces to be selected for AlertmanagerConfig discovery. If nil, only check own namespace. | *[metav1.LabelSelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#labelselector-v1-meta) | false |
 | minReadySeconds | Minimum number of seconds for which a newly created pod should be ready without any of its container crashing for it to be considered available. Defaults to 0 (pod will be considered available as soon as it is ready) This is an alpha field and requires enabling StatefulSetMinReadySeconds feature gate. | *uint32 | false |
+| globalAlertmanagerConfig | GlobalAlertmanagerConfig used to set the global alertmanager config. The specified global alertmanager config will not force add a namespace label in routes and inhibitRules. | *[GlobalAlertmanagerConfig](#globalalertmanagerconfig) | false |
```

Comment:
Additionally I'd lean towards the inclusion of a standalone section in https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/alerting.md perhaps to which defines and describes the behaviours and the use case?